rke/k8s_rke_system_images.go

```diff
@@ -8902,6 +8902,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.22.10",
 		},
+		// Enabled in Rancher v2.7.4
+		"v1.26.4-rancher2-1": {
+			Etcd:                      "rancher/mirrored-coreos-etcd:v3.5.6",
+			Kubernetes:                "rancher/hyperkube:v1.26.4-rancher2",
+			Alpine:                    "rancher/rke-tools:v0.1.89",
+			NginxProxy:                "rancher/rke-tools:v0.1.89",
+			CertDownloader:            "rancher/rke-tools:v0.1.89",
+			KubernetesServicesSidecar: "rancher/rke-tools:v0.1.89",
+			KubeDNS:                   "rancher/mirrored-k8s-dns-kube-dns:1.22.8",
+			DNSmasq:                   "rancher/mirrored-k8s-dns-dnsmasq-nanny:1.22.8",
+			KubeDNSSidecar:            "rancher/mirrored-k8s-dns-sidecar:1.22.8",
+			KubeDNSAutoscaler:         "rancher/mirrored-cluster-proportional-autoscaler:1.8.6",
+			Flannel:                   "rancher/mirrored-flannelcni-flannel:v0.21.4",
+			FlannelCNI:                "rancher/flannel-cni:v0.3.0-rancher7",
+			CalicoNode:                "rancher/mirrored-calico-node:v3.24.1",
```

Comment:
Everything else is at `v3.25.0`, is there a reason for CalicoNode to be at `v3.24.1`? https://github.com/rancher/image-mirror/blob/master/images-list#L823

Reply:
left it by mistake. fixed.